src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,63 @@
+import { React, useState } from "react";
+import BREAKPOINT from "../../../variables/breakpoint";
+import COLOR from "../../../variables/color";
+import FONT_FAMILY from "../../../variables/font_family";
+import TEXTS from "../../../variables/texts";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton";
+import Task from "../../Molecules/Task";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList([...taskList, { name: "", initializing: true }]);
+  };
+
+  const onTaskComplete = (index) => {
+    const taskListCpy = [...taskList];
+    setTaskList(taskListCpy.filter((element, num) => num !== index));
```

Comment:
```suggestion
    setTaskList(taskList.filter((element, idx) => idx !== index));
```
`filter` は非破壊的メソッドなので、配列をコピーする意味は薄いかと思います。
参考：https://typescriptbook.jp/reference/values-types-variables/array/array-operations#%E7%A0%B4%E5%A3%8A%E7%9A%84%E3%81%AA%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89

`index` という変数名が被っている場合は `idx` と略すケースを見たことがあります。
もしくは、引数 `index` として受け取っているところを `indexOfCompletedTask` とかにして受け取るのも個人的にはありだと思います。

```suggestion
    const filteredTaskList = taskList.filter((element, idx) => idx !== index)
    setTaskList(filteredTaskList);
```
ごちゃごちゃして見づらいと思ったらこうして一時変数に入れるのもありですね